### ISSUE TITLE
Crashes with basic usage

### ISSUE THREAD

**Issue by octavz** [aigisuk/terraform-kubernetes-argocd#16]
Hi, trying to use this module but I  have some issues. Could you help? Thank you.

I have a `.tf` file with this content:

```
module "argocd" {
  source  = "aigisuk/argocd/kubernetes"
  version = "0.2.6"
}
```

trying to `terraform apply` crashes with this:

```
module.argocd.helm_release.argocd: Creating...
╷
│ Error: template: argo-cd/templates/NOTES.txt:31:7: executing "argo-cd/templates/NOTES.txt" at <has "--insecure" .Values.server.extraArgs>: error calling has: Cannot find has on type string
│
│   with module.argocd.helm_release.argocd,
│   on .terraform/modules/argocd/chart.tf line 2, in resource "helm_release" "argocd":
│    2: resource "helm_release" "argocd" {
```

**Comment by octavz** [comment]
I am on:

```
Terraform v1.4.0
on linux_amd64
+ provider registry.terraform.io/hashicorp/google v4.57.0
+ provider registry.terraform.io/hashicorp/google-beta v4.57.0
+ provider registry.terraform.io/hashicorp/helm v2.9.0
+ provider registry.terraform.io/hashicorp/kubernetes v2.18.1
+ provider registry.terraform.io/hashicorp/random v3.4.3

Your version of Terraform is out of date! The latest version
is 1.4.2. You can update by downloading from https://www.terraform.io/downloads.html
```

**Comment by colinwilson** [comment]
@octavz Thanks for this. 👍

`server.extraArgs."--insecure"` has been deprecated. I've updated the module to use the `configs.params.server.insecure` value instead. Please try release [`0.2.7`](https://github.com/aigisuk/terraform-kubernetes-argocd/releases/tag/0.2.7)

**Comment by colinwilson** [comment]
Fixed in release [`0.2.7`](https://github.com/aigisuk/terraform-kubernetes-argocd/releases/tag/0.2.7)
